### DISCUSSION TITLE
[wrong java -cp][hw1]

### DISCUSSION THREAD

**Discussion by kopter0** [prosyslab-classroom/cs348-information-security#732] [Q&A]
In PDF it is said to use 
`java -cp pad_oracle.jar:bcprov-jdk15-130.jar:python_interface_v1_2.jar python_interface_v1_2`

however seems like semicolons should be used instead of colon

and be 

`java -cp pad_oracle.jar;bcprov-jdk15-130.jar;python_interface_v1_2.jar python_interface_v1_2`

it doesnt work with first and work with second, so I think it is better to change pdf

**Comment by Lim-Sangho** [comment]
Zshell on my mac works with the first one, not with the second. Perhaps it depends on the environments.

**Comment by pacokwon** [comment]
I think it does ![depend on the environment](https://en.wikipedia.org/wiki/Classpath).
It says that for windows you'll need to use `;` and for Unix-like environments you'll need to use `:`
